src/Appwrite/Migration/Version/V19.php

```diff
@@ -640,6 +646,21 @@ private function getFunctionCommands(Document $function): string
         return $commands;
     }
 
+    private function migrateEnumAttributeSize(): void
+    {
+        \Co\run(function () {
+            foreach ($this->documentsIterator('attributes') as $attribute) {
+                go(function ($attribute) {
+                    $attribute->setAttribute('size', Database::LENGTH_KEY);
+                    $this->projectDB->updateDocument('attributes', $attribute->getId(), $attribute);
+                    $databaseInternalId = $attribute->getAttribute('databaseInternalId');
+                    $collectionInternalId = $attribute->getAttribute('collectionInternalId');
+                    $this->changeAttributeInternalType('database_' . $databaseInternalId . '_collection_' . $collectionInternalId, $attribute->getAttribute('key'), 'varchar(255)');
```

Comment:
It would be good to avoid adapter specific things when possible. Can we use [`$this->projectDB->updateAttribute()`](https://github.com/utopia-php/database/blob/60a61d00570bb66e2277be9c88b947279c729623/src/Database/Database.php#L1310) instead?

Reply:
what is the adapter specific thing here?

Reply:
Changed to updateAttribute